src/lib/services/user-service.ts

```diff
@@ -400,11 +402,19 @@ class UserService {
         if (!receiver) {
             throw new NotFoundError(`Could not find ${receiverEmail}`);
         }
+        if (this.passwordResetTimeouts[receiver.id]) {
+            return;
```

Comment:
Works as a bugfix for now. More suggested improvements in: https://linear.app/unleash/issue/1-245/improve-forgotten-password-api